istioctl/pkg/writer/ztunnel/configdump/configdump.go

```diff
@@ -51,7 +51,7 @@ func (c *ConfigWriter) PrintBootstrapDump(outputFormat string) error {
 
 func (c *ConfigWriter) PrintFullSummary() error {
 	_, _ = c.Stdout.Write([]byte("\n"))
-	if err := c.PrintWorkloadSummary(WorkloadFilter{}); err != nil {
+	if err := c.PrintWorkloadSummary(WorkloadFilter{Verbose: true}); err != nil {
```

Comment:
QQ - is this not something that can be toggled by supplying a `--verbose` flag?

Should it be a toggle, rather than hardcoded (eventually)?

Reply:
Ideally I would say yes, control this with flags.  I think that's a good longer term solution but right now we don't have a verbose flag and the default behavior of the workload portion of ztunnel-config defaults to this verbose output anyhow so this just matches the normal behavior